# This is a rebase for the 5.10.129 release.
# Original commit:  f85b140f08ad704091af612e6abdc2d32def89b9
# URL:  https://github.com/google/bbr/commit/f85b140f08ad704091af612e6abdc2d32def89b9

License header:

net/ipv4/tcp_rate.c:
// SPDX-License-Identifier: GPL-2.0-only

--- a/net/ipv4/tcp_rate.c.orig2	2022-07-12 15:54:56.654462717 -0700
+++ b/net/ipv4/tcp_rate.c	2022-07-12 15:55:34.514955713 -0700
@@ -109,6 +109,7 @@ void tcp_rate_skb_delivered(struct sock
 		rs->prior_mstamp     = scb->tx.delivered_mstamp;
 		rs->is_app_limited   = scb->tx.is_app_limited;
 		rs->is_retrans	     = scb->sacked & TCPCB_RETRANS;
+		rs->tx_in_flight     = scb->tx.in_flight;
 		rs->last_end_seq     = scb->end_seq;
 
 		/* Record send time of most recently ACKed packet: */
